week1/homework/src/urlExtension.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
```

Comment:
Please don't disable Eslint, instead try to fix the lint errors/warnings if exist.